build-logic/conventions/src/main/kotlin/SerializationPlugin.kt

```
@@ -0,0 +1,20 @@
+    override fun apply(target: Project) {
+        with(target) {
+            with(pluginManager) {
+                apply("org.jetbrains.kotlin.plugin.serialization")
+            }
+
+            dependencies {
+                implementation(libs.library("kotlinx.serialization.json"))
+            }
+        }
+    }
+}
```

detekt (1.23.8)

[warning] The file does not contain a package declaration.

(detekt.potential-bugs.MissingPackageDeclaration)